### DISCUSSION TITLE
OpenShift : Stuck on Waiting for Kustomization "flux-system/flux-system" to be reconciled

### DISCUSSION THREAD

**Discussion by CarolynAnn** [fluxcd/flux2#1410] [Q&A] [answered]
I'm running the flux command to bootstrap my OpenShift cluster. I'm trying to watch for images added to a docker registry. I have a public repository. 
The command I'm running is 
```
flux bootstrap github --owner=<USERNAME> --repository=<REPO> --branch=main --personal --components-extra=image-reflector-controller,image-automation-controller
```
However, I'm running into the issue
```
◎ waiting for Kustomization "flux-system/flux-system" to be reconciled
✗ context deadline exceeded
```
When I check my OpenShift events, it looks like the controllers are failing to create
```
Error creating: pods "source-controller-xxx-" is forbidden: unable to validate against any security context constraint: [provider restricted: .spec.securityContext.fsGroup: Invalid value: []int64{1337}: 1337 is not an allowed group]
```
Has anyone else seen this before?

**Comment by hiddeco** [comment, marked as answer]
See #1051

**Comment by stefanprodan** [comment]
@CarolynAnn please see the OpenShift documentation https://fluxcd.io/docs/use-cases/openshift/#security-context-constraints

**Comment by CarolynAnn** [comment]
Thank you @hiddeco and @stefanprodan!
